pkg/kapp/clusterapply/waiting_changes.go

```diff
@@ -13,9 +14,10 @@ import (
 )
 
 type WaitingChangesOpts struct {
-	Timeout       time.Duration
-	CheckInterval time.Duration
-	Concurrency   int
+	Timeout             time.Duration
+	ResourceWaitTimeout time.Duration
```

Comment:
ResourceWaitTimeout -> ResourceTimeout (we are already under waiting struct and the other field is just called Timeout)

Reply:
Done